### PR TITLE
Add Prometeus and Grafana

### LINKED ISSUE #2: Docker-compose use Postgresql with Prometheus and Grafana
Implementing Prometheus monitoring for PostgreSQL involves setting up Prometheus, a PostgreSQL exporter, and integrating it into your environment. Below is a comprehensive list of requirements and steps to achieve this:

Requirements for Implementing Prometheus for PostgreSQL

Software Requirements
Prometheus: The core monitoring system to collect metrics.
PostgreSQL Exporter: A Prometheus-compatible exporter to fetch PostgreSQL metrics.
Popular options include the [postgres_exporter](https://github.com/prometheus-community/postgres_exporter).
PostgreSQL: A running PostgreSQL database instance.
https://prometheus.io/docs/prometheus/latest/installation/

### PR DESCRIPTION
**Pull request by KrystianKlik**
Adding Grafana and Prometheus to the project, synchronizing them with the database, and integrating everything using Docker Compose.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+version: '3.8'
```
Comment (KrystianKlik) [inline docker-compose.yaml:1]: It's obsolete, delete it

File: docker-compose.yaml [hunk 2]
```diff
@@ -11,9 +13,64 @@ services:
...
+      - GF_SECURITY_ADMIN_USER=admin
+      - GF_SECURITY_ADMIN_PASSWORD=admin
+    ports:
+      - "3000:3000"
+    networks:
+      - monitoring
+    depends_on:
+      - prometheus
+
+networks:
```
Comment (KrystianKlik) [inline docker-compose.yaml:71]: You can delete networks use everywhere, because `bridge` is automatically used so it's redundant

**Review by KrystianKlik — COMMENTED** [review]
image

you don't store dashboards, have you thought about preserving them somehow or it's going to be in the next PR?

**Review by antypowy1998 — APPROVED** [review]
Ok

**Review by antypowy1998 — APPROVED** [review]

**Review by antypowy1998 — APPROVED** [review]

**Review by antypowy1998 — APPROVED** [review]
ok

**Comment by antypowy1998** [comment]
ok

**Pull request merged**